runtime/executor/method.cpp

```diff
@@ -626,7 +627,7 @@ Error Method::init(executorch_flatbuffer::ExecutionPlan* s_plan) {
 
     for (size_t i = 0; i < n_delegate; ++i) {
       const auto& delegate = *delegates->Get(i);
-      BackendInitContext backend_init_context(method_allocator);
+      BackendInitContext backend_init_context(method_allocator, method_name_);
```

Comment:
Multiple delegates will have the same method_name_, if all the delegates are in this method. Is it right?

Reply:
The delegate blob contains multimethod, here the backend knows what the top level method is called.